src/library_html5.js

```diff
@@ -27,10 +27,12 @@ var LibraryJSEvents = {
     // so that we can report information about that element in the event message.
     previousFullscreenElement: null,
 
+#if MIN_IE_VERSION != TARGET_NOT_SUPPORTED || MIN_SAFARI_VERSION <= 8 || MIN_EDGE_VERSION <= 12 || MIN_CHROME_VERSION <= 21 // https://caniuse.com/#search=movementX
```

Comment:
comment here says movementX but the variables are for screenX?

https://caniuse.com/#search=screenX says the compatibility for those is different. for example desktop safari has no support at any version.

Reply:
Hmm that is just how movementX emulation is implemented: it is computed from screenX/Y variables: `var movementX = e.screenX-JSEvents.previousScreenX;`. (If screenX is not supported, then computing movementX via the emulation path becomes much harder, but we have not accounted for that scenario before this PR either)

Reply:
Oh also, you are looking at "Touch events: screenX" there. The appropriate one to look at is [MouseEvent API: screenX](https://caniuse.com/#feat=mdn-api_mouseevent_screenx), which is ubiquitous.

Reply:
Oh sorry then, my mistake.